feat: added options to playground

screensaversclub opened the pull request:
Added some options to try out in the playground 

- replacement character
- remove characters (regex) 
- lowercase 
- trim leading and trailing characters

Still brainstorming how best to do the char maps.

Trott commented on index.html:
```diff
@@ -6,17 +6,51 @@
+
+			<fieldset>
+
+				<label for="replacement">Replacement char</label>
+				<input type="text" id="replacement" name="replacement" maxlength="1" />
```
Does it make sense to put the default value as placeholder text for this and other configuration fields?

Trott commented on playground.js:
```diff
@@ -0,0 +1,40 @@
+
+  const opts = {};
+
+  if (replacement.length === 1) {
```
Muti-character replacements are supported.

```js
slug('a b c', {replacement: '///'}) // 'a///b///c'
```

So maybe this instead?

```suggestion
  if (replacement.length > 0) {
```

Trott commented:
Thank you for doing this! It needs to happen if the playground is going to be at all useful.

screensaversclub commented:
> Would you be OK breaking the `remove` RegExp stuff into a separate follow-on pull request, or at least it's own commit? Regular expression munging can be buggy and complicated. (Maybe it isn't in this case, but I'd prefer to evaluate it separately anyway.) In contrast, `replacement`, `lower`, `trim`, and `fallback` should all be very straightforward to implement.

@Trott  Definitely happy to do so. Just sanity check, how would you approach something like this from a git perspective? 

I'm thinking to do a new branch from `main`, then manually apply the changes from `playground-options` but I'm wondering if there's a smarter way to do that.

screensaversclub commented:
@Trott  I have removed the `remove` regex code, to be committed separately into another PR. 

- Added `fallback` option which I somehow missed.  
- Added a default `input` value as well, feel free to suggest something else (maybe something with specific characters)!

screensaversclub commented:
@Trott  sorry for the multiple PRs - I'm proposing to close this one as I did a squash to simplify the git history for just the basic playground options. That'll be on #438 . The plan is to then introduce the `remove` option branching off of that commit but on a separate PR.
